test/parallel/test-child-process-muted-array-proto.js

```diff
@@ -0,0 +1,28 @@
+'use strict';
+
+const common = require('../common');
+const assert = require('node:assert');
+const { spawn } = require('node:child_process');
+
+// The UNIX version of this test is enough.
+if (common.isWindows) {
+  common.skip('This test is enough for Unix-like systems');
+  return;
+}
+
+// Refs: https://github.com/nodejs/node/issues/55932
```

Comment:
In addition to the link, a short description of the issue being tested would be helpful so folks don't have to actually follow the link to see what this is about.